content/metadata.yaml

```
@@ -149,6 +152,7 @@ authors:
     email: gitter@biostat.wisc.edu
     contributions:
       - Methodology
+      - Software
```

```suggestion
      - Software
      - Project Administration
```
@agitter I think this should be here too, I always struggle to map the CRediT taxonomy onto real life, but it feels like the people who are going through the PRs as they come in and coaching people through that process are all doing PA (which to date is primarily you, Christian, Nils and me)

---

Thanks @rando2, that makes sense.  I agree that the CRediT taxonomy categories are subjective and hard to map precisely to different contributors.  You're doing a good job of monitoring contributions and applying them consistently to different contributors.